packages/core/core/src/factories/useUserFactory.ts

```diff
@@ -44,13 +48,14 @@ export const useUserFactory = <USER, UPDATE_USER_PARAMS, REGISTER_USER_PARAMS ex
 
     const register = async ({ user: providedUser }) => {
       Logger.debug('useUserFactory.register', providedUser);
+      resetErrorValue();
 
       try {
         loading.value = true;
         error.value.register = null;
         user.value = await factoryParams.register(context, providedUser);
       } catch (err) {
-        error.value.register = err;
+        error.value.register = err.response?.data || err;
```

Comment:
please assign only `err` you can't assume there is always data field, each integration can be different and this is core (agnostic)

Reply:
hmm use that mapping here: https://github.com/vuestorefront/vue-storefront/blob/release-2-2/packages/commercetools/composables/src/useUser/factoryParams.ts#L41

wrap authenticate in try-catch, and in the catch statement throw the error again but with mapping err.response?.data.errors (I guess errors are there)

Reply:
I found out that `err.response?.data` wasn't needed.